### PR TITLE
Training: I deleted every single part of capture

### PR DESCRIPTION
**Pull request by CollinAvidano**
closes #1180

### CODE REVIEW

File: soccer/gameplay/skills/capture.py [hunk 1]
```diff
@@ -1,270 +0,0 @@
...
-
-    # The distance to avoid the  ball in coarse approach
-    CoarseApproachAvoidBall = 0.1
-
-    # Minimum speed (On top of ball speed) to move towards the ball
-    FineApproachMinDeltaSpeed = 0.1
-
-    # Proportional term on the distance error between ball and robot during fine approach
-    # Adds to the fine approach speed
-    FineApproachDistanceMultiplier = .1
```
Comment (JNeiger) [inline soccer/gameplay/skills/capture.py]: rip

**Comment by tjones320** [comment]
Why are we spending time on this and not the coach play?

**Comment by JNeiger** [comment]
Killed since this was a demo

File: soccer/gameplay/skills/capture.py [hunk 2]
```diff
@@ -1,270 +0,0 @@
...
-
-    def on_enter_coarse_approach(self):
-        self.lastApproachTarget == None
-
-    def execute_coarse_approach(self):
-        pos = self.find_intercept_point()
-
-        # Don't move our target if it's somewhat close to our old target
-        if (self.lastApproachTarget != None and
-            (pos - self.lastApproachTarget).mag() < 0.1):
```
Comment (jgkamat) [inline soccer/gameplay/skills/capture.py]: Next time, please refactor this magic number to a constant before deleting it please

**Pull request closed without merging**